test/helper.js

```diff
@@ -37,11 +40,12 @@ helper.hasPulseCredentials = function() {
 helper.hasAzureCredentials = function() {
   return cfg.app.hasOwnProperty('azureAccounts') && cfg.app.azureAccounts;
 };
-
 // Configure PulseTestReceiver
 if (cfg.pulse.password) {
   helper.events = new testing.PulseTestReceiver(cfg.pulse, mocha);
-};
+} else {
+  helper.events = new testing.PulseTestReceiver({fake:true}, mocha);
+}
```

Comment:
@djmitche Oh. So since helper.events is removed, then the waitFor method by the helper.events will also be deleted and it does not matter since we also use fake pulse in this case and thus it wont take a while to return, is it?

Reply:
correct!